openwisp_controller/config/tests/test_api.py

```diff
@@ -510,3 +512,20 @@ def test_vpn_delete_api(self):
         r = self.client.delete(path)
         self.assertEqual(r.status_code, 204)
         self.assertEqual(Vpn.objects.count(), 0)
+
+    def test_get_request_with_change_perm(self):
```

Comment:
Have wrote test for only change perm, should I also add test with view perm. Since this tests are almost similar to the tests wrote while implementing this feature so skipped it.

Reply:
yes please, view perm as well, a tiny bit of redundancy for more protection is a good compromise

Reply:
Done adding. :+1: